recipe/meta.yaml

```
@@ -33,6 +32,13 @@ requirements:
     - pyyaml >=5.3.1
     - toolz >=0.10.0
     - importlib_metadata >=4.13.0
+  run_constrained:
```

Should this constraint be added (like in [the `dask` package]( https://github.com/conda-forge/dask-feedstock/blob/b01e7863385d797924969b3ddb79a66d14463ac7/recipe/meta.yaml#L25 ))? If so, should [this also be updated upstream]( https://github.com/dask/dask/blob/499f4055da707fa76d06a2b79f408f124eee4723/pyproject.toml#L48 )?

```suggestion
  run_constrained:
    - cytoolz >=0.11.0
```